pages/osx/autopkg

```diff
@@ -0,0 +1,29 @@
+# autopkg
+
+> Automation framework for macOS software packaging and distribution.
+
+- Run autopkg recipe:
+
+`autopkg run {{recipe_identifier}}`
+
+- Run autopkg with in verbose mode:
+
+`autopkg run -vvv {{recipe_identifier}}`
+
+- Run autopkg with specified pkg:
+
+`autopkg run {{recipe_identifier}} --pkg {{/path/to/downloaded/pkg_or_dmg}}`
+
+- Update one or more recipe repo:
+
+`autopkg repo-update {{repo_names}}`
+
+`autopkg repo-update all`
```

Comment:
Why not give this one a description aswell? Something like this:
```
- Update all repos:

`autopkg repo-update all`
```

This is again something that would've been picked up by `tldr-lint`.